packages/round-manager/src/features/round/__tests__/RoundDetailForm.test.tsx

```diff
@@ -129,121 +128,164 @@ describe("<RoundDetailForm />", () => {
     const submitButton = screen.getByRole("button", {
       name: /next|launch/i,
     });
-    const supportSelection = screen.getByTestId("support-type-select");
+
+    const supportSelection = screen.getByTestId("roundMetadata.support.type-testid");
     fireEvent.click(supportSelection);
-    const firstSupportOption = screen.getAllByTestId("support-type-option")[0];
-    fireEvent.click(firstSupportOption);
+
+    const emailSupportOption = screen.getByTestId("roundMetadata.support.type-option-Email");
```

Comment:
It would be good if we could avoid using test ids unless we cannot avoid it, in this case I see we have labels, can we not use those?

Reply:
I don't think it's a personal preference, the testing library is designed around the idea that tests behave like users, so we're testing the UI as close as to how it's used, and test IDs go against that completely, users don't see test IDs, they see buttons, links, labels, text etc. That's how tests should query the DOM. Test IDs should only be used in a pinch for very specific cases.

I recommend going through the testing library documentation: https://testing-library.com/docs/guiding-principles

This article summarizes the approach pretty well: https://derekndavis.com/posts/getbytestid-overused-react-testing-library